Suspicious code, possibly copy & paste error

The following code looks suspicious:

```
//its important to test lengths so tests don't panic and exit early
if len(alice) != 1 || len(alice) != 1 || len(alice) != 1 || len(alice) != 1 {
    t.Error("Unexpected data size returned")
}
```

and

```
//its important to test lengths so tests don't panic and exit early
if len(alice) != 2 || len(alice) != 2 || len(alice) != 2 || len(alice) != 2 {
    t.Error("Unexpected data size returned")
}
```

```
/src/github.com/square/inspect/mysql/tools/tools_test.go:255:5: identical expressions on the left and right side of the '||' operator
/src/github.com/square/inspect/mysql/tools/tools_test.go:286:5: identical expressions on the left and right side of the '||' operator
```

--- a/cmd/inspect/main.go
+++ b/cmd/inspect/main.go
@@ -64,7 +64,7 @@ func main() {
 		err := termui.Init()
 		termui.UseTheme("helloworld")
 		if err != nil {
-			log.Fatalf("Unable to initialize termui", err)
+			log.Fatal("Unable to initialize termui:", err)
 		}
 		widgets = uiWidgets()
 		uiSummaryBody = uiSummary(widgets)

--- a/mysql/tools/tools_test.go
+++ b/mysql/tools/tools_test.go
@@ -252,7 +252,7 @@ func TestQueryMapFirstColumnToRow1(t *testing.T) {
 		t.Error("Unexpected data returned")
 	}
 	//its important to test lengths so tests don't panic and exit early
-	if len(alice) != 1 || len(alice) != 1 || len(alice) != 1 || len(alice) != 1 {
+	if len(alice) != 1 || len(bob) != 1 || len(charlie) != 1 || len(david) != 1 {
 		t.Error("Unexpected data size returned")
 	}
 	if alice[0] != "Jan" || bob[0] != "Feb" ||
@@ -283,7 +283,7 @@ func TestQueryMapFirstColumnToRow2(t *testing.T) {
 		t.Error("Unexpected data returned")
 	}
 	//its important to test lengths so tests don't panic and exit early
-	if len(alice) != 2 || len(alice) != 2 || len(alice) != 2 || len(alice) != 2 {
+	if len(alice) != 2 || len(bob) != 2 || len(charlie) != 2 || len(david) != 2 {
 		t.Error("Unexpected data size returned")
 	}
 	if alice[0] != "Jan" || bob[0] != "Feb" ||
@@ -317,7 +317,7 @@ func TestBadConnection1(t *testing.T) {
 
 	_, _, err = testdb.QueryDb("SELECT * FROM people;")
 	if err != nil {
-		t.Error("failed to reconnect: %v", err)
+		t.Errorf("failed to reconnect: %v", err)
 	}
 }
 

